README.md

```diff
@@ -109,7 +109,7 @@ Tokens may yield a `bool` to indicate successful execution of a transfer, may `r
 
 Tokens should be able to execute normal EIP-20 transfers under all circumstances, and submissions that would concern this capability (i.e. blacklists, pausability, transfers of `0`, etc.) are not considered in-scope.
 
-The system should be able to support tokens of varying decimal configurations as long as they are within sensible bounds (i.e. $0 \le decimals \le 18$) and miniscule value truncations **in favour of the protocol / at the expense of the user** are considered acceptable.
+The system should be able to support tokens of varying decimal configurations as long as they are within sensible bounds (i.e. $0 \lt decimals \le 18$) and minuscule value truncations **in favour of the protocol / at the expense of the user** are considered acceptable.
```

Comment:
I think minuscule is incorrect and miniscule is correct.  As much as I like ladybugs, I don't think they are involved in the math.